src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -109,10 +109,8 @@ def separate_vgs
               CheckBox(
                 Id(:separate_vgs),
                 format(
-                  # TRANSLATORS: %{widget_label} refers to the label of the widget. %{paths} is a
-                  # comma separated list of paths
-                  _("%{widget_label}\n(%{paths})"),
-                  widget_label: WIDGET_LABELS[:use_separate_vgs],
+                  "%{widget_label}\n(%{paths})",
```

Comment:
I miss the translator comment here, but since previous PR is already approved LGTM

Reply:
That's the point, no translation there. The string is marked in [line 36](https://github.com/yast/yast-storage-ng/blob/bf516775e588caae0fe5076d4417f0f93f9a1e5f/src/lib/y2storage/dialogs/guided_setup/select_scheme.rb#L36).